public/js/encryptionManager.js

```diff
@@ -151,38 +168,34 @@ EncryptionManager.prototype.getKeyManager = function getKeyManager(data, callbac
 
   console.log("[ENCRYPTION MANAGER] (getKeyManager) Starting KeyManager creation with privateKey: "+privateKey+" publicKey: "+publicKey+" passphrase: "+passphrase);
   console.log("[ENCRYPTION MANAGER] (getKeyManager) Starting KeyManager creation");
+
   kbpgp.KeyManager.import_from_armored_pgp({
-    armored: publicKey
+    armored: privateKey
   }, function(err, keyManager) {
-    if (!err) {
-      keyManager.merge_pgp_private({
-        armored: privateKey
+    if (err) {
+      return callback(err);
+    }
+    if (keyManager.is_pgp_locked()) {
+      keyManager.unlock_pgp({
+        passphrase: passphrase
       }, function(err) {
-        if (!err) {
-          if (keyManager.is_pgp_locked()) {
-            keyManager.unlock_pgp({
-              passphrase: passphrase
-            }, function(err) {
-              if (err) { return callback(err) };
-              keyManager.sign({}, function(err) {
-                if (err) { return callback(err) };
-                console.log("Loaded private key with passphrase");
-                return callback(err, keyManager);
-              });
-            });
-          } else {
-            console.log("Loaded private key w/o passphrase");
-            return callback(err, keyManager);
-          }
-        } else {
-          return callback(err, null);
```

Comment:
Here you were importing the public and then merging the private. This is not necessary so I removed the bloat. This is still a little bloaty as we have several places now that we are loading keys using kbpgp directly. We should break out this kind of thing into its own function and call it where necessary.